### ISSUE TITLE
[V3 Components] Opt-out of V3 components

### ISSUE THREAD

**Issue by FranECross** [department-of-veterans-affairs/va.gov-cms#16976]
## Description
The Design team will be migrating all V1 components to V3 components on February 16th, which doesn't provide the Public Websites team adequate time to implement V3.

This ticket is to implement opting out of V3 components so that we can prioritize the tickets implementing V3 for components currently using V1.

For products not using any components, we'll prioritize those at a later date, or as those products are being enhanced/changed in the future.

### V1 components that need to be opted out of
- Accordion: Used in
   - CLPs
	- Benefits Landing pages
	- Benefits Detail Pages
	- Income Limits
	- PACT Act
	- FAQ page
	- Resources & Support Detail page
- Additional Info
	- Income Limits
	- PACT Act
- Alert: Used in
	- Benefits Landing pages
	- Benefits Detail pages
	- Income Limits
	- Discharge Upgrade
	- Global Search Results
	- Find a Form
	- Resources & Support Detail page
	- FAQ page
	- BTSSS login widget
- Button: Used in
	- Income Limits
	- PACT Act
- Button Pair: Used in
	- Income Limits
	- PACT Act
- Number Input: Used in
	- Income Limits
- Radio Button: Used in
	- Discharge Upgrade
- Select: Used in
	- Income Limits
	- Discharge Upgrade
	- Find a Form
- Modal: Used in
	- Find a Form
- Pagination: Used in
	- Resources and Support Results
	- Events
- Search Input: Used in
	- Homepage
	- Find a Form

**For reference, The other v3 components that exist but I do not believe we are using their v1 equivalents anywhere at this time are:**
- Breadcrumbs
- Featured Content
- Checkbox
- Memorable date
- Privacy Agreement
- Progress bar - Segmented
- Statement of Truth
- Text input
- Textarea
- Process List

## Documentation
* [Sitewide V3 DS component review spreadsheet](https://docs.google.com/spreadsheets/d/1A7mQMnL2d5JcDKHZ2daVYQVr85qOljrwM0aUk_inO88/edit#gid=0)
* Summary of components with V3 correlates: https://docs.google.com/spreadsheets/d/1A7mQMnL2d5JcDKHZ2daVYQVr85qOljrwM0aUk_inO88/edit#gid=1489013257

### User story

**AS A** PO/PM for Public Website products
**I WANT** to opt-out of the V3 component migration for some or all of our products currently using V1 (final list TBD after assessment)
**SO THAT** we have time to switch from V1 to V3

## Acceptance criteria

- [ ] All V1 components with V3 correlates are opted out of the V3 update on Feb 16th.

**Comment by randimays** [comment]
Status update: the vets-website PR has been merged and I am verifying in staging. I am waiting on a PW review for the content-build portion so I can merge that into staging. Hoping to get both pieces to production today.

**Comment by randimays** [comment]
content-build has been merged into prod 👍🏻 vets-website is having some CI issues and many merges have not yet made it to prod since yesterday. I'm waiting on that to resolve to do a quick validation and call this done.

**Comment by randimays** [comment]
This was held up for a couple days while Platform triaged the ongoing CI issues with vets-website. They were able to resolve them yesterday afternoon, so the v3 opt-outs for vets-website are in production and good to go! Closing!
